docs/google.rst

```diff
@@ -126,6 +126,8 @@ google.enableRequesterPaysBuckets              When ``true`` uses the configured
 google.batch.allowedLocations                  Define the set of allowed locations for VMs to be provisioned. See `Google documentation <https://cloud.google.com/batch/docs/reference/rest/v1/projects.locations.jobs#locationpolicy>`_ for details (default: no restriction. Requires version ``22.12.0-edge`` or later).
 google.batch.bootDiskSize                      Set the size of the virtual machine boot disk, e.g ``50.GB`` (default: none).
 google.batch.cpuPlatform                       Set the minimum CPU Platform, e.g. ``'Intel Skylake'``. See `Specifying a minimum CPU Platform for VM instances <https://cloud.google.com/compute/docs/instances/specify-min-cpu-platform#specifications>`_ (default: none).
+google.batch.instanceTemplate                  Set the Compute Engine instance template to create a job (default: none).
+google.batch.installGpuDrivers                  When ``true`` install the GPU drivers to the virtual machine (default: ``false``).
```

Comment:
This option shouldn't be exposed to users, it should be set automatically by Nextflow if the `accelerator` directive was used

Reply:
If we only use an `accelerator ` option to add GPUs to an instance, we don't need an `installGpuDrivers` option. However, if we support `instanceTemplate` option, we need the one. Because there is no way to know a created instance template has GPUs or not. It totally depends how a user creates an instance template and we only know the name of the instance template. Hence, if we support an instance template option, an `installGpuDrivers` option is inevitable.

Reply:
Okay, I will just add a note that this option is only needed when using an instance template.